mava/wrappers/jumanji.py

```diff
@@ -62,3 +55,87 @@ def observation_spec(self) -> specs.Spec[Observation]:
             "step_count",
         )
         return self._env.observation_spec().replace(step_count=step_count)
+
+
+class RWAREWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the Robotic Warehouse environment."""
+
+    def __init__(self, env: RobotWarehouse):
+        super().__init__(env)
+
+    def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
+        """Modify the timestep for the Robotic Warehouse environment."""
+        n_agents = self._env.num_agents
+        observation = Observation(
+            agents_view=timestep.observation.agents_view,
+            action_mask=timestep.observation.action_mask,
+            step_count=jnp.repeat(timestep.observation.step_count, n_agents),
+        )
+        reward = jnp.repeat(timestep.reward, n_agents)
+        discount = jnp.repeat(timestep.discount, n_agents)
+        return timestep.replace(observation=observation, reward=reward, discount=discount)
+
+
+class LBFWrapper(MultiAgentWrapper):
+    """
+     Multi-agent wrapper for the Level-Based Foraging environment.
+
+    Args:
+        env (Environment): The base environment.
+        aggregate_rewards (bool): If True, aggregates rewards across agents.
+        reward_aggregation_function (str): The function for aggregating rewards ("sum" or "mean").
+    """
+
+    def __init__(
+        self,
+        env: LevelBasedForaging,
+        aggregate_rewards: bool = False,
+        reward_aggregation_function: str = "sum",
+    ):
+        super().__init__(env)
+        self._env: LevelBasedForaging
+        self._aggregate_rewards = aggregate_rewards
+        self.reward_aggregation_function = reward_aggregation_function
+        self._num_agents = self._env.num_agents
+        # Mapping aggregation functions
+        self.aggregation_functions: Dict[str, Callable] = {
+            "sum": jnp.sum,
```

Comment:
I agree with @sash-a's previous comment on this. I think we should just do what the benchmarking paper does and keep that as our default aggregation function. The only boolean we would then need is whether or not to aggregate the rewards over agents or not.

Reply:
I'll change the aggregation to true the default  to use the sum of the individual rewards as the benchmarking paper did and make the other case optional (remove the mean function as well).